server/services/github/graphQL.ts

```diff
@@ -0,0 +1,93 @@
+import { CommitSummary } from "../../types/CommitSummary";
+import { getEnv } from "../../utils/getEnv";
+
+/**
+ * Query builder is here: https://docs.github.com/en/graphql/overview/explorer
+ */
+const buildGraphQuery = (owner: string, repo: string): string => {
+  const query = `
+  {
+    repository(owner: "${owner}", name: "${repo}") {
```

Comment:
so is this still one query per [owner/repo] ?
then we still need a lot of queries?

Reply:
previously we were doing one separate call for every single commit, so we're about 2 orders of magnitude better :)

but yes there will still be a few queries

it MAY be possible to stack all our needs into a single graph query, but there are to the way the lookups work, so if you're including a ton of old repos in your query you'll get loads of old commits in the response. You can't (or don't seem to be able to) filter down the query by date within the graph query